app/lua/lnodemcu.c

```diff
@@ -0,0 +1,116 @@
+/*
+** $Id: ltm.c,v 2.8.1.1 2007/12/27 13:02:25 roberto Exp $
+** Tag methods
+** See Copyright Notice in lua.h
+*/
+
+
+#define lnodemcu_c
+#define LUA_CORE
+
+#include "lua.h"
+#include <string.h>
+
+#include "lobject.h"
+#include "lstate.h"
+#include "lauxlib.h"
+#include "lgc.h"
+#include "lstring.h"
+#include "ltable.h"
+#include "ltm.h"
+#include "lrotable.h"
+#include "platform.h"
+
+extern int debug_errorfb (lua_State *L);
+#if 0
+extern int pipe_create(lua_State *L);
+extern int pipe_read(lua_State *L);
+extern int pipe_unread(lua_State *L);
+extern int pipe_write(lua_State *L);
+#endif
+/*
+** Error Reporting Task.  We can't pass a string parameter to the error reporter
+** directly through the task interface the call is wrapped in a C closure with
+** the error string as an Upval and this is posted to call the Lua reporter.
+*/
+static int report_traceback (lua_State *L) {
+// **Temp**  lua_rawgeti(L, LUA_REGISTRYINDEX, G(L)->error_reporter);
```

Comment:
Nope placeholder because on the next API, I am adding an extra `node` API call to allow the app to set the error reporter CB.  What these changes do is to send PANIC tracebacks to stdout, but on a prod system the app developer might want to log them or send them back to a central syslog.